### ISSUE TITLE
check_player_priv crash

### ISSUE THREAD

**Issue by thomasrudin** [pandorabox-io/pandorabox.io#284]
```
minetest_1           | 2019-08-18 18:37:53: ERROR[Main]: ServerError: AsyncErr: ServerThread::run Lua: Runtime error from mod 'default' in callback LuaABM::trigger(): Runtime error from mod 'default' in callback node_on_dig(): .../minetest/games/minetest_game/mods/default/functions.lua:572: core.check_player_pri
minetest_1           | 2019-08-18 18:37:53: ERROR[Main]: vs expects a player or playername as argument.
minetest_1           | 2019-08-18 18:37:53: ERROR[Main]: stack traceback:
minetest_1           | 2019-08-18 18:37:53: ERROR[Main]: 	[C]: in function 'error'
minetest_1           | 2019-08-18 18:37:53: ERROR[Main]: 	/usr/local/share/minetest/builtin/game/misc.lua:11: in function 'check_player_privs'
minetest_1           | 2019-08-18 18:37:53: ERROR[Main]: 	.../minetest/games/minetest_game/mods/default/functions.lua:572: in function 'can_interact_with_node'
minetest_1           | 2019-08-18 18:37:53: ERROR[Main]: 	...are/minetest/games/minetest_game/mods/default/chests.lua:102: in function 'can_dig'
minetest_1           | 2019-08-18 18:37:53: ERROR[Main]: 	/usr/local/share/minetest/builtin/game/item.lua:543: in function </usr/local/share/minetest/builtin/game/item.lua:538>
minetest_1           | 2019-08-18 18:37:53: ERROR[Main]: 	[C]: in function 'dig_node'
minetest_1           | 2019-08-18 18:37:53: ERROR[Main]: 	/data/world//worldmods/technic/technic/radiation.lua:510: in function 'old_action'
minetest_1           | 2019-08-18 18:37:53: ERROR[Main]: 	/data/world//worldmods/monitoring/builtin/abm_calls.lua:17: in function </data/world//worldmods/monitoring/builtin/abm_calls.lua:9>
minetest_1           | 2019-08-18 18:37:53: ERROR[Main]: stack traceback:
minetest_1           | 2019-08-18 18:37:53: ERROR[Main]: 	[C]: in function 'dig_node'
minetest_1           | 2019-08-18 18:37:53: ERROR[Main]: 	/data/world//worldmods/technic/technic/radiation.lua:510: in function 'old_action'
minetest_1           | 2019-08-18 18:37:53: ERROR[Main]: 	/data/world//worldmods/monitoring/builtin/abm_calls.lua:17: in function </data/world//worldmods/monitoring/builtin/abm_calls.lua:9>

```

**Comment by coil0** [comment]
Flowing corium on locked chest causes this when it attempts to dig the chest.

This is fixed by minetest/minetest_game#2448.

**Comment by thomasrudin** [comment]
should be fixed in mt game 5.1
